src/presentation/http/router/noteSettings.ts

```diff
@@ -79,14 +82,19 @@ const NoteSettingsRouter: FastifyPluginCallback<NoteSettingsRouterOptions> = (fa
 
     const noteSettings = await noteSettingsService.getNoteSettingsByNoteId(noteId);
 
+    const team = await noteSettingsService.getTeamWithUsersInfoByNoteId(noteId);
```

Comment:
"team" is a part of noteSettings, so it's better to add this field at the `noteSettingsService.getNoteSettingsByNoteId`